.github/workflows/test.yml

```diff
@@ -464,30 +354,17 @@ jobs:
   unit_test_admin:
     name: Unit tests - Admin Interface
     runs-on: ubuntu-latest
-    needs: [build_test_admin]
     steps:
       ##########################################################################
       # CHECKOUT CODE ##########################################################
       ##########################################################################
       - name: Checkout code
         uses: actions/checkout@v3
       ##########################################################################
-      # DOWNLOAD DOCKER IMAGES #################################################
-      ##########################################################################
-      - name: Download Docker Image (Admin Interface)
-        uses: actions/download-artifact@v3
-        with:
-          name: docker-admin-test
-          path: /tmp
-      - name: Load Docker Image
-        run: docker load < /tmp/admin.tar
-      ##########################################################################
       # UNIT TESTS ADMIN INTERFACE #############################################
       ##########################################################################
       - name: Admin Interface | Unit tests
-        run: |
-          docker run -v ~/coverage:/app/coverage --rm gradido/admin:test yarn run test
-          cp -r ~/coverage ./coverage
+        run: cd admin && yarn && yarn run test
       ##########################################################################
       # COVERAGE CHECK ADMIN INTERFACE #########################################
       ##########################################################################
```

Comment:
missing change on `result_path: ./coverage/lcov.info`

Reply:
Strange, this result path has been changed in [this commit](https://github.com/gradido/gradido/pull/2719/commits/f5c44e8ca3b40c6e85cb5f3e3a26bcdfee257052).